google/cloud/spanner/internal/session.h

```diff
@@ -29,7 +29,11 @@ namespace spanner_internal {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
 /**
- * A class that represents a Session.
+ * Sessions are a central concept in the Spanner Cloud API: all Spanner
+ * reads/writes are performed through a session. A session must be created
+ * before any non-admin operation can be performed. Once created, a session
+ * persists until it is destroyed by the client, or is reclaimed due to
```

Comment:
nit: s/is reclaimed/reclaimed/

or use "it is reclaimed"

Reply:
Done.